pip-dep/requirements.txt

```diff
@@ -17,3 +17,4 @@ torch~=1.4.0
 tornado==4.5.3
 websocket_client~=0.57.0
 websockets~=8.1.0
+git+https://github.com/gmuraru/CrypTen.git@2aa0c2f14b472a8df6a85be1cfc4630479a10b2f#egg=crypten
```

Comment:
This should be removed after merging the [Fix plan](https://github.com/OpenMined/PySyft/pull/3422) changes